main.go

```diff
@@ -101,9 +101,22 @@ func runWithOptions(allOpts *options, remaining []string) error {
 
 	opts := allOpts.OutputOpts
 	if opts.Raw {
+		if opts.File != "torch.svg" {
+			torchlog.Print("Printing raw flamegraph input to", opts.File)
+			f, err := os.Create(opts.File)
```

Comment:
can we use `ioutil.WriteFile` here and avoid the extra noise from create/write/close?